docker-compose.yml

```diff
@@ -1,21 +1,21 @@
 version: '3.7'
 services:
   networkml:
-    image: '${TOOL_NAMESPACE}/networkml:v0.6.0'
+    image: 'iqtlabs/networkml:v0.6.0'
```

Comment:
@davedittrich looks like an accidental revert here?

Reply:
It think that was a mistake in my original commit before I realized that `networkml` and `pcapplot` are in their own repos. The current `develop` branch should have the right values.

https://github.com/IQTLabs/packet_cafe/compare/master...davedittrich:develop